src/agent/agent.c

```diff
@@ -1212,6 +1212,14 @@ static int agent_method_stop_dep(sd_bus_message *m, void *userdata, UNUSED sd_bu
         return 0;
 }
 
+/*************************************************************************
+ ************** org.containers.hirte.Agent.Reload ***************
+ ************************************************************************/
+
+static int agent_method_reload(UNUSED sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bus_error *ret_error) {
+        return 1;
```

Comment:
please correct me if I am wrong, but it seems a stub function? What's 1 meant here having a bunch of parameters in the function?  Maybe create a macro saying 'NOT_IMPLEMENTED_YET` (just example).